### PR TITLE
refactor: move deleted tauri APIs, prepare for next release

### PR DESCRIPTION
**Pull request by lucasfernog**
Keeping up with https://github.com/tauri-apps/tauri/pull/6925

### CODE REVIEW

File: plugins/app/src/commands.rs [hunk 1]
```diff
@@ -12,8 +12,7 @@ pub fn name<R: Runtime>(app: AppHandle<R>) -> String {
 
 #[tauri::command]
 pub fn tauri_version() -> &'static str {
-    // TODO: return actual tauri version with `tauri::VERSION`
-    env!("CARGO_PKG_VERSION")
+    tauri::VERSION
```
Comment (amrbashir) [inline plugins/app/src/commands.rs:15]: should we change this to use the newly added `tauri::version()` in dev? https://github.com/tauri-apps/tauri/commit/7c237209207cd2938df660b6fd87d3b7d728bd03

**Review by amrbashir — APPROVED** [review]

File: plugins/app/src/commands.rs (hunk 1 above)
Comment (amrbashir) [reply plugins/app/src/commands.rs:15]: or should we change the api in `dev` (still unreleased) to a `const` variable like `next`?

Comment (lucasfernog) [reply plugins/app/src/commands.rs:15]: It should be a const unless we parse the version string to semver::Version IMO

Comment (amrbashir) [reply plugins/app/src/commands.rs:15]: cool, I will update the `dev` branch then

**Pull request merged**